src/main/kotlin/skytils/skytilsmod/features/impl/handlers/SpamHider.kt

```diff
@@ -380,6 +417,42 @@ class SpamHider {
                 }
             }
 
+            // Sea creature catch
+            if (SEA_CREATURES.contains(unformatted)) {
+                when (Skytils.config.scCatchHider) {
+                    1, 2 -> cancelChatPacket(event, Skytils.config.scCatchHider == 2)
+                    else -> {
+                    }
+                }
+            }
+
+            // Legendary sea creature catch
+            if (LEGENDARY_SEA_CREATURES.contains(unformatted)) {
+                when (Skytils.config.legendaryScCatchHider) {
+                    1, 2 -> cancelChatPacket(event, Skytils.config.legendaryScCatchHider == 2)
+                    else -> {
+                    }
+                }
+            }
+
+            // Good catch
```

Comment:
Could be changed to use formatted but doesn't really matter.

Reply:
The only issue with using unformatted + contains is that it can trigger on player chat, even though it's unlikely

Reply:
Since contains checks against the set in this case it should only trigger if the full chat message is exactly that phrase which a player wouldn't be able to send.